### DISCUSSION TITLE
RaspberryMatic CCU Addon startet nicht in HA

### DISCUSSION THREAD

**Discussion by ZdravecG** [OpenCCU/OpenCCU#2703] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Nach einem Restore von HA fehlte das RaspberryMatic CCU addon. Ich habe es neu installiert, um homematic backup einzuspielen. Leider startet das Addon nicht. Folgende Meldung wird angezeigt, start add-on zeigt "The add-on is starting, this can take some time..." für einige Minuten und dann wird wieder die Meldung gezeigt, dass das Addon nicht gestartet ist.
![image](https://github.com/jens-maus/RaspberryMatic/assets/116811557/30a5eee3-b0c3-4c6e-adb8-f91e641e9be6)

### Describe the behavior you expected

Add on sollte starten

### Steps to reproduce the issue

1. RaspberryMatic CCU starten

### What is the version this bug report is based on?

3.75.6.20240316

### Which base platform are you running?

ha-addon (HomeAssistant Add-on)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
Mounting /data as /usr/local (Home Assistant Add-On): OK
Identifying host system: Oracle Corporation VirtualBox (oci), OK
Initializing RTC Clock: onboard, OK
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Running seedrng: OK
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.5, OK
Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:00:06.0-2, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:00:06.0-2, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.18, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: creating new SSL cert... OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: .......OK
Starting rfd: .OK
Starting HMIPServer: ........OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: OK
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.75.6.20240316 (raspmatic_oci_amd64)
```

### Additional information

Vielleicht liegt der Fehler schon daran, dass das HA restore das Addon nicht installiert hat, aber es gab keine Fehlermeldungen in der Logdatei.

**Comment by jens-maus** [comment]
Na startet das Addon jetzt oder nicht, weil du ja eine Logausgabe zeigst die klar zeigt das das Addon ja anscheinend neustartet. IMHO Sieht das für mich nach einem HA Problem aus und ist kein generelles Problem/Bug von/in RaspberryMatic. Ergo gehört das eher in den Diskussionsteil um nach Hilfe zu suchen...

**Comment by ZdravecG** [comment]
Im Logdatei steht, dass der Addon gestartet ist, aber im UI kommt immer die Meldung, dass es nicht läuft. 
Im Protokoll finde Folgende Meldung

````
Logger: hahomematic.central
Quelle: /usr/local/lib/python3.12/site-packages/hahomematic/central/__init__.py:505
Erstmals aufgetreten: 12:41:16 (51 Vorkommnisse)
Zuletzt protokolliert: 12:59:59

GET_LOCAL_IP: Can't resolve host for de838cd8-raspberrymatic
````

**Reply by jens-maus** [reply]
Das liest sich in der Tat nach einem reinen HA Problem. Ergo, kompletter HA Neustart machen und weiter diesbzgl. analysieren. RaspberryMatic kann an der stelle nix machen...

**Comment by Baxxy13** [comment, marked as answer]
Nach einem Restore von HA sollte der HA komplett neu gestartet werden, also nicht nur HA sondern der Host/VM.

Das Problem ist bekannt, war den HA-Entwicklern aber wohl nicht relevant genug um darauf zu reagieren bzw. etwas dagegen zu unternehmen.
Siehe: https://github.com/home-assistant/supervisor/issues/4773

**Reply by ZdravecG** [reply]
Danke, das hat geholfen.
